stix2/test/conftest.py

```diff
@@ -13,12 +12,12 @@
 @pytest.fixture
 def clock(monkeypatch):
 
-    class mydatetime(dt.datetime):
+    class mydatetime(stix2.utils.STIXdatetime):
```

Comment:
There might be easier ways to patch our custom datetime subclass. I remember some of these shenanigans were due to the fact that `dt.datetime` is built in. We can leave this as is, though, since it should work.